images/nodejs/Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM amazonlinux:2
+FROM public.ecr.aws/amazonlinux/amazonlinux:2023
 
-RUN yum install -q -y libstdc++.i686 glibc.i686 tar gzip xz shadow-utils && \
```

Comment:
The fact that 32bit libraries were being pulled here should likely make everybody quite sad.

Reply:
My recollection is that getting the node image created in a way that aligned with our OSS processes was problematic with AL2, though I could have just been missing something. I wasn't happy with the result but wanted to get it out the door.

Reply:
AL2023 is the first Amazon Linux version to package nodejs, so grabbing the binaries from nodejs.org that were somewhat generic was always going to be a bit of "fun" in ensuring continued compatibility.